src/main/java/ch/admin/bag/covidcertificate/sdk/core/extensions/VaccinationEntryExtensions.kt

```diff
@@ -55,13 +56,8 @@ fun VaccinationEntry.vaccineDate(): LocalDateTime? {
 	if (this.vaccinationDate.isEmpty()) {
 		return null
 	}
```

Comment:
Some of the functions in the Vacc/Test/RecovEntryExtensions check for emptyness, others don't. I think these checks can be removed since the parsing fails for empty strings (and the try-catch of the util will return `null`).